__tests__/app.test.js

```diff
@@ -580,4 +580,32 @@ describe("GET /api/articles ADDITIONAL FEATURE - Filer by topic", () => {
         );
       });
   });
-});
\ No newline at end of file
+});
+
+describe("GET /api/articles/:articleid ADDITIONAL FEATURE - Filer by topic", () => {
```

Comment:
This is ticket 12 which is adding comment count to the endpoint. Please change the test name to reflect this 👍